data/Publications.json

```diff
@@ -7258,5 +7258,59 @@
         "bibcode": "2021ApJ...911....7Z",
         "doi": null,
         "description": null
+    },
+    {
+        "reference": "WFAU19",
+        "bibcode": "noADS_WFAU19",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "Schn23b",
+        "bibcode": "2023AJ....166..103S",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "UCS_24",
+        "bibcode": "noADS_UCS_24",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "usyn24",
+        "bibcode": "noADS_usyn24",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "McMa21",
+        "bibcode": "2021yCat.2367....0M",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "Nowa20",
+        "bibcode": "2020A&A...642L...2N",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "Stol20b",
+        "bibcode": "2020A&A...644A..13S",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "Stol21",
+        "bibcode": "2021AJ....162..286S",
+        "doi": null,
+        "description": null
+    },
+    {
+        "reference": "Alle16b",
+        "bibcode": "2016PhDT.......189A",
+        "doi": null,
+        "description": null
```

Comment:
These should have DOIs and descriptions. Did you use the `ingest_publication` function? You might need to setup your ADS Token.